src/routers/Stripe/index.ts

```diff
@@ -0,0 +1,202 @@
+import assert from 'assert';
+import { Router } from 'express';
+import { WhpptRequest } from 'src';
+import { calculateTotal, getStripCustomerIdFromContact, loadOrder } from './common';
+
+const router = Router();
+
+export type StripeRouterConstructor = () => Router;
+
+const stripe = require('stripe')(process.env.STRIPE_KEY);
```

Comment:
We need to get this reading from config. Will be different for every client. 
OK for now but we do need a todo

Reply:
Added todo